fedora/client/openidbaseclient.py

```diff
@@ -129,7 +133,9 @@ def __init__(self, base_url, login_url=None, useragent=None, debug=False,
         :kwarg timeout: A float describing the timeout of the connection. The
             timeout only affects the connection process itself, not the
             downloading of the response body. Defaults to 120 seconds.
-
+        :kwarg retry_backoff_factor: Backoff factor to apply between attempts.
```

Comment:
If neither `retries` nor `retry_backoff_factor` can be adjusted after object initialization, could you perhaps mention it in the docstring? It might be confusing otherwise. Our usual approach was:
```
bodhi = Bodhi2Client()
bodhi.retries = 5
```
That won't work anymore, and it could be quite hard to guess why without an explanation.